### DISCUSSION TITLE
Resource capacity limit doesn't work

### DISCUSSION THREAD

**Discussion by ieovigo** [LibreBooking/librebooking#511] [Q&A] [answered]
Hi,

I have installed librebooking 2.8.6.1 on debian 12 server and we have a problem, the system does not allow us to make more than one reservation at the same time for any resource, even if we set unlimited capacity or for example 2. It shows a message indicating that there is a reservation conflict.

What could be the problem?

Thank you so much

**Comment by ieovigo** [comment, marked as answer]
Hello,

I think I already found the error. It has been a problem of interpretation of the concept of resource capacity.

We thought it was concurrent booking capability, for example, allowing 4 people to book a room at the same time, because that room has 4 computers.

I think that to control that, you have to create 4 resources, one for each computer.

it's right?

Thank you

**Comment by effgarces** [comment]
Hi, exactly. In fact that was one of the things that first tripped me when  I started using Booked (now Librebooking). Resource capacity here, is geared toward the capacity of a room, so by taking advantage of the fact that you can invite people to an event you can ensure that you don't exceed the capacity of the room.
Besides creating 4 resources,  there's also a another trick that you can do, create a single resource, that can be booked simultaneously by many then  create an accessory with the total quantity of 4 and associate it with the resource, make the minimum quantity that you can reserve 1, that way, when booking you will need to add at least one of that accessory in order to progress, when all accessories are booked it will no longer be possible to book the resource.

**Comment by Martin35415** [comment]
Hi, but the resource planning doesn't seem to work properly.

Example:
I schedule an appointment with 25 people (Participant List or Invitees? What's the difference?), one resource is a person who has a capacity of 12 people, I add this resource. I can still schedule the appointment - that makes no sense. What does the capacity specification for a resource refer to? It can't be an accessory, it has to be people, as it is intended.

If I have to assign an accessory to a resource and then have to manually add the people to this accessory in the booking process for the appointment so that the limit applies, that is far too much work. 

The adjustment that the capacity (
This resource has a capacity of X people) is checked based on the participants would be very important for me to be able to use this tool - can you drive the development of this feature forward? I would also be prepared to pay a certain amount of money for this.
